src/launcher.c

```diff
@@ -791,10 +812,16 @@ int main(int argc, char *argv[])
         sdk_version != SDK_API_LEVEL_8 && sdk_version != SDK_API_LEVEL_9 &&
         sdk_version != SDK_API_LEVEL_10 && sdk_version != SDK_API_LEVEL_11 &&
         sdk_version != SDK_API_LEVEL_12 && sdk_version != SDK_API_LEVEL_13 &&
-        sdk_version != SDK_API_LEVEL_14 && sdk_version != SDK_API_LEVEL_15) {
+        sdk_version != SDK_API_LEVEL_14 && sdk_version != SDK_API_LEVEL_15 &&
+        sdk_version != SDK_API_LEVEL_18) {
```

Comment:
Not sure that support Stax on API_LEVEL_18 is relevant. It can be handy for internal tests but as of now there is no Stax release on this API_LEVEL.